dist/components/list/_list.scss

```diff
@@ -0,0 +1,62 @@
+// List
+// ===
+//
+// Style list as a list
+
+
+// Component Variables
+// ---
+
+$list__margin: if(variable_exists(v-space), $v-space 0, 10px 0) !default;
+$list__padding-left: 15px !default;
+$list__item-margin: if(variable_exists(v-space), $v-space/2 0, 5px 0) !default;
+
+
+// List
+// ---
+//
+// 1. Keep bullets on the same vertical line as content
+
+.c-list {
+    margin: $list__margin;
+    padding-left: $list__padding-left; // 1
+
+
+    // Unordered list
+    // ---
+
+    &.c--ul {
+
+        list-style-type: disc;
+
+        & & {
+            list-style: circle;
```

Comment:
Not sure if this is what we want as a default. Can we let the designer on each project opt into this rather than opt out?